CONTRIBUTING.md

```diff
@@ -97,19 +97,20 @@ to work on.
 
 # 7. Turn coffee and documentation into code and documentation!
 
-Synapse's code style is documented [here](docs/code_style.md). Please follow
-it, including the conventions for the [sample configuration
-file](docs/code_style.md#configuration-file-format).
-
-There is a growing amount of documentation located in the [docs](docs)
-directory. This documentation is intended primarily for sysadmins running their
+There is a growing amount of documentation located in the [`docs`](docs)
+directory, with a rendered version [available online](https://matrix-org.github.io/synapse).
+This documentation is intended primarily for sysadmins running their
 own Synapse instance, as well as developers interacting externally with
-Synapse. [docs/dev](docs/dev) exists primarily to house documentation for
-Synapse developers. [docs/admin_api](docs/admin_api) houses documentation
+Synapse. [`docs/development`](docs/development) exists primarily to house documentation for
+Synapse developers. [`docs/admin_api`](docs/admin_api) houses documentation
 regarding Synapse's Admin API, which is used mostly by sysadmins and external
 service developers.
 
-If you add new files added to either of these folders, please use [GitHub-Flavoured
+Synapse's code style is documented [here](docs/code_style.md). Please follow
+it, including the conventions for the [sample configuration
+file](docs/code_style.md#configuration-file-format).
```

Comment:
I think it'd be better to link to the rendered documentation than to markdown files directly; in general that's where we prefer to send people.

Reply:
Hmm, I think that might need some care: [contributing_guide.md](https://github.com/matrix-org/synapse/blob/dmr/link-to-rendered-docs/docs/development/contributing_guide.md) includes CONTRIBUTING.md directly.

Could move CONTRIBUTING's contents into contributing_guide and have CONTRIBUTING just say "see the rendered documentation".

Reply:
Note that https://github.com/matrix-org/synapse/pull/10599#pullrequestreview-730678221 is in agreement.

Reply:
Encouraging! Doing this now.